app/views/shared/_navbar.html.erb

```diff
@@ -37,12 +36,14 @@
             <% end %>
             <li class="dropdown">
               <a class="dropdown-toggle" aria-expanded="true" role="button" data-toggle="dropdown" href="#">
-                <%= t('language') %>
+                <%= image_tag("file.#{locale}.png") %>
```

Comment:
I18n, http://guides.rubyonrails.org/i18n.html, It dynamically recognizes the page language